.github/workflows/test.yml

```diff
@@ -0,0 +1,35 @@
+name: test
+on: [push, pull_request]
+
+jobs:
+  e2e-test:
+    runs-on: ubuntu-24.04
```

Comment:
@siburu  Can you use `ubuntu-22.04` same as lcp and lcp-go?https://github.com/datachainlab/lcp/blob/main/.github/workflows/test.yml
(Also, note that we currently use the intel sgx sdk is for 22.04 in the test)

Reply:
@bluele yes, done!